pages/Static.vue

```diff
@@ -1,167 +1,151 @@
-          link: getPathForStaticPage('/customer-service'),
-          component: StaticShortExample
-        },
-        {
-          title: i18n.t('Store locator'),
-          link: localizedRoute('/store-locator'),
-          component: StaticExample
-        },
-        {
-          title: i18n.t('Delivery'),
-          link: '/delivery',
-          component: StaticShortExample
-        },
-        {
-          title: i18n.t('Return policy'),
-          link: '/returns',
-          component: StaticExample
-        },
-        {
-          title: i18n.t('Privacy policy'),
-          link: '/privacy',
-          component: StaticShortExample
+          title: 'Privacy Policy',
+          message: `To help us achieve our goal of providing the highest quality products and
+      services, we use information from our interactions with you and other
+      customers, as well as from other parties. Because we respect your privacy,
+      we have implemented procedures to ensure that your personal information is
+      handled in a safe, secure, and responsible manner. We have posted this
+      privacy policy in order to explain our information collection practices
+      and the choices you have about the way information is collected and used.`
         },
         {
-          title: i18n.t('Size guide'),
-          link: '/size-guide',
-          component: StaticExample
+          title: 'Security',
+          message: `Personal information provided on the website and online credit card
+      transactions are transmitted through a secure server. We are committed to
+      handling your personal information with high standards of information
+      security. We take appropriate physical, electronic, and administrative
+      steps to maintain the security and accuracy of personally identifiable
+      information we collect, including limiting the number of people who have
+      physical access to our database servers, as well as employing electronic
+      security systems and password protections that guard against unauthorized
+      access.`
         },
         {
-          title: i18n.t('Contact us'),
-          link: '/contact',
-          component: StaticShortExample
+          title: 'Additional Information',
+          message: `This website ("website") is operated by Luma Inc., which includes Luma
+      stores, and Luma Private Sales. This privacy policy only covers
+      information collected at this website, and does not cover any information
+      collected offline by Luma. All Luma websites are covered by this privacy
+      policy.`
         }
       ]
     };
   },
   computed: {
-    activeComponent () {
-      const matchedNav = this.navigation.find(
-        nav => nav.link === this.$route.path
-      );
-      return matchedNav ? matchedNav.component : null;
+    breadcrumbs () {
+      return [
+        { text: i18n.t('Homepage'), route: { link: '/' } },
+        { text: this.currentRoute.title }
```

Comment:
here https://github.com/DivanteLtd/vsf-capybara/pull/86/files?file-filters%5B%5D=.csv&file-filters%5B%5D=.js&file-filters%5B%5D=.vue&hide-deleted-files=true#diff-5b84d70689800171aa0f199fd4a791a1R8 it requires to exist `route.link`. It doesn't break? :P

Reply:
It works fine and it does not report any `"TypeError: breadcrumb.route is undefined"` because this is the last item in the breadcrumbs array. The `SfBradcrumbs` requires `route.link` for all breadcrumb items [except the last one](https://github.com/DivanteLtd/storefront-ui/blob/d362cf68516b8f64955931645da52cc771499d82/packages/vue/src/components/atoms/SfBreadcrumbs/SfBreadcrumbs.vue#L10). For last breadcrumb item only `text` is required.

Reply:
ok, because last one is not a link but only a text